### PR TITLE
Pypi restructure

### PR DESCRIPTION
**Pull request by arporter**
Closes issue #3.

### CODE REVIEW

File: doc/system_specific_setup.rst [hunk 1]
```diff
@@ -135,10 +138,11 @@ Install pytest for running python tests
...
-::
+You can now run the PSyclone python tests:
+
+.. parsed-literal::
 
-   > cd PSyclone_trunk/src/tests
+   > cd PSyclone-\ |release|\ /src/psyclone/tests
    > py.test
 
 Documentation
```
Comment (rupertford) [inline doc/system_specific_setup.rst]: It is not possible to build the documentation without first installing using setup.py as this creates the version numbers. The documentation needs to be updated (as does the doc README file).

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Science and Technology Facilities Council
+# However, it has been created with the help of the GungHo Consortium,
```
Comment (rupertford) [inline setup.py]: please remove gungho consortium reference

File: src/__init__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Science and Technology Facilities Council
+# However, it has been created with the help of the GungHo Consortium,
```
Comment (rupertford) [inline src/__init__.py]: please remove gungho consortium words

File: src/psyclone/__init__.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Science and Technology Facilities Council
+# However, it has been created with the help of the GungHo Consortium,
```
Comment (rupertford) [inline src/psyclone/__init__.py]: please remove gungho consortium words

File: src/psyclone/gen_kernel_stub.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Science and Technology Facilities Council
+# However, it has been created with the help of the GungHo Consortium,
```
Comment (rupertford) [inline src/psyclone/gen_kernel_stub.py]: please remove reference to gungho consortium

File: README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+PSyclone from the Python Package Index (https://pypi.python.org/pypi).
+That being so, you can install this copy of PSyclone by doing:
+
+    $ python setup.py install
+
+or, if you have pip:
+
+    $ pip install .
+
+The latter attempts to perform a system-wide install. For a user-local
```
Comment (arporter) [inline README.md]: So it does. Corrected/added more text.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+installation simply add the --user flag:
+
+    $ pip install --user .
```
Comment (arporter) [inline README.md]: Done.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+There is also a simple installer (contributions/install) which creates a
```
Comment (arporter) [inline README.md]: She did. I didn't think I'd requested a review yet :-)

File: doc/conf.py [hunk 9]
```diff
@@ -41,16 +41,17 @@
...
 #
-# The short X.Y version.
-version = '1.4'
-# The full version, including alpha/beta/rc tags.
-release = '1.4.1'
+# We use the version module generated by setup.py so that we are always
+# consistent with the version specified in that file.
+# 'version' is the short X.Y version and 'release' is the full version,
+# including any alpha/beta/rc tags.
+from psyclone.version import version as release, short_version as version
```
Comment (arporter) [inline doc/conf.py]: Yes. Will do so once branch is merged back to master.

File: doc/system_specific_setup.rst [hunk 10]
```diff
@@ -28,21 +28,24 @@ Change directory to where you would like to place the code (where
...
-::
+   > wget https://github.com/stfc/PSyclone/archive/\ |release|\ .tar.gz
+   > gunzip \ |release|\ .tar.gz
+   > tar xf \ |release|\ .tar
+   > rm \ |release|\ .tar
+
+Set your python path and path appropriately:
 
-   > cd PSyclone-1.4.1
+.. parsed-litera::
```
Comment (arporter) [inline doc/system_specific_setup.rst]: Well spotted!

File: doc/system_specific_setup.rst (hunk 1 above)
Comment (arporter) [inline doc/system_specific_setup.rst]: Good point. Fixed both the documentation and the readme.

**Comment by arporter** [comment]
All requested changes have been made. Ready for second review by @rupertford.

**Pull request merged**